move_group:
  arm:
    # Velocity and acceleration scaling for manipulator planning in MoveIt
    planning_time: 50
    planning_attempts: 5
    velocity_scaling: 0.2
    acceleration_scaling: 0.99

    left: [0.791303, -0.109673, -1.007676, -2.054506, 1.608512, 2.168921, -1.934002]
    right: [-0.849273, -0.996814, 1.041194, -2.477326, 1.713547, 2.817316, -1.667401]
    home: [0.040662, -0.398527, 0.192162, -1.813924, 0.020539, 1.639937, 0.230023]


  panda_arm:
    # Velocity and acceleration scaling for manipulator pla nning in MoveIt
    planning_time: 50
    planning_attempts: 5
    velocity_scaling: 0.2
    acceleration_scaling: 0.99

    # Home joint positions for the panda robot
    # home: [0.0, -0.785, 0.0, -2.356, 0.0, 1.571, 0.785]
    home: [-0.000076, -0.715166, 0.000062, -2.275858, 1.569985, 2.316859, 0.000053]
  hand:
    planning_time: 50
    planning_attempts: 5
    velocity_scaling: 0.2
    acceleration_scaling: 0.99

    # Home joint positions for the panda robot
    home: [0.0]

  gripper:
    planning_time: 50
    planning_attempts: 5
    velocity_scaling: 0.2
    acceleration_scaling: 0.99

    # Home joint positions for the panda robot
    home: [0.04]